redshift-connector.gemspec

```diff
@@ -16,10 +16,10 @@ Gem::Specification.new do |s|
   s.require_path = 'lib'
 
   s.required_ruby_version = '>= 2.1.0'
-  s.add_dependency 'activerecord', '< 5'
-  s.add_dependency 'activerecord4-redshift-adapter'
+  s.add_dependency 'activerecord'
+  s.add_dependency 'activerecord-redshift'
   s.add_dependency 'redshift-connector-data_file', '~> 1.0.0'
-  s.add_dependency 'pg', '~> 0.18.0'
+  s.add_dependency 'pg'
```

Comment:
pg version is needed because pg 0.19 only supports PostgreSQL 9, and it is (logically) incompatible with Redshift.  It does NOT mean that pg 0.19 does not work with Redshift, but it is not confirmed.